tests/unit/createApp/realizeCustomElements.ts

```diff
@@ -531,6 +542,26 @@ registerSuite({
 		}
 	},
 
+	'<app-projector data-css-transition> attribute': {
+		'is false if empty'() {
+			return app.realize(root).then(() => {
+				assert.isTrue(projectorSpy.calledWith( match({ cssTransitions: false })));
```

Comment:
> any particular reason?

Wait you can't read my mind? 😞 

It's partly because I find it too hard to memorize Sinon's assertion APIs. But also because AVA uses `power-assert` and I tend to write boolean expressions as my test assertions.